modules/administration/pages/virtual-hosts.adoc

```diff
@@ -107,8 +107,16 @@ image::systems_virtual_host_managers_vmware.png[scaledwidth=80%]
 
 [IMPORTANT]
 ====
-If you do not refresh the data from a new Virtual Host Manager, host data will not be inventoried and therefore will not be displayed under menu:Systems[Virtual Systems].
+If you do not refresh the data from a new Virtual Host Manager, host data will not be inventoried and therefore will not be displayed under menu:System List[Virtual Systems].
 ====
 +
 
-. View inventoried ESXi host objects and resources by selecting menu:Systems[Virtual Systems] .
+. View inventoried ESXi host objects and resources by selecting menu:System List[Virtual Systems].
+
+[NOTE]
+====
+Depending on your environment and settings, you might receive an ``invalid certificate`` error when you connect to the ESXi server from a web browser using HTTPS.
+If this occurs, refreshing the data from the virtual hosts server will fail.
```

Comment:
We could simply remove "Depending on your environment and settings, ", because it does not bring much information and is not even really true.

To be very honest, it will not "fail" with a message, but silently provide no result... The failure can be seen only in the logs. But it's a product problem, not a documentation issue. So unrelated to this PR.